cmd/posix-errors.go

```diff
@@ -36,8 +35,7 @@ func isSysErrNoSys(err error) bool {
 // Not supported error
 func isSysErrOpNotSupported(err error) bool {
 	if pathErr, ok := err.(*os.PathError); ok {
-		switch pathErr.Err {
-		case syscall.EOPNOTSUPP:
+		if pathErr.Err == syscall.EOPNOTSUPP {
```

Comment:
Further simplification is possible simply do `return pathErr.Err == syscall.EOPNOTSUPP`